### PR TITLE
Add release notes for crash log location

### PR DESCRIPTION
**Pull request by wolfs**
See #12345

### CODE REVIEW

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -40,6 +40,38 @@ details of 2
 ## n
 -->
 
+## Show location of Java crash log
+
+Sometimes, the Gradle daemon crashes due to some bugs in Java, Gradle or some third party plugin.
+When Java crashes, it writes a [fatal error log](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/felog.html) to the current working directory.
+Since it may be hard to find out what the working directory of the daemon process is, Gradle now prints the location of the crash log to the console.
+For example, the following build crashed, and creates a crash log at `/home/user/project/hs_err_pid11783.log`:
```
Comment (lptr) [inline subprojects/docs/src/docs/release/notes.md]:
```suggestion
For example, the following build crashed, and created a crash log at `/home/user/project/hs_err_pid11783.log`: 
```

File: subprojects/docs/src/docs/release/notes.md [hunk 2]
```diff
@@ -40,6 +40,38 @@ details of 2
 ## n
 -->
 
+## Show location of Java crash log
+
+Sometimes, the Gradle daemon crashes due to some bugs in Java, Gradle or some third party plugin.
```
Comment (lptr) [inline subprojects/docs/src/docs/release/notes.md]:
```suggestion
Sometimes the Java virtual machine executing the Gradle daemon crashes due to a bug in Java, Gradle itself, or in the user code being executed.
```

File: subprojects/docs/src/docs/release/notes.md [hunk 3]
```diff
@@ -40,6 +40,38 @@ details of 2
...
+When Java crashes, it writes a [fatal error log](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/felog.html) to the current working directory.
```
Comment (lptr) [inline subprojects/docs/src/docs/release/notes.md]:
```suggestion
When the Java VM crashes, a [fatal error log](https://docs.oracle.com/javase/8/docs/technotes/guides/troubleshoot/felog.html) is created in the working directory of the daemon process.
```

Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md]: Should we link the Java 8 docs here, or something newer perhaps?

Comment (wolfs) [reply subprojects/docs/src/docs/release/notes.md]: I am linking to the Java 13 docs now.

I also applied your suggestion manually.

**Comment by wolfs** [comment]
@lptr PTAL!

File: subprojects/docs/src/docs/release/notes.md (hunk 3 above)
Comment (lptr) [reply subprojects/docs/src/docs/release/notes.md]: I guess that's fine, though maybe we should link to LTE versions only? Probably nitpicking.

Comment (wolfs) [reply subprojects/docs/src/docs/release/notes.md]: Java 11 it is.

**Pull request merged**